### ISSUE TITLE
When is the Checkpoint Release?

### ISSUE THREAD

**Issue by miyajiyuta** [tomchen-ctj/OST#1]
Thank you for your amazing work!
I would love to try out your model. 
Could you let me know when you plan to release the model? 
Additionally, how much VRAM would be required for training?

Thank you!

**Comment by tomchen-ctj** [comment]
Hi @miyajiyuta , thanks for your interest in our work!

> Could you let me know when you plan to release the model?

We plan to release model checkpoints this month, so please stay tuned!
> Additionally, how much VRAM would be required for training?

We use 8 Nvidia 3090 in our basic training settings, and because OST does not need to maintain all the categories' logits in every training step, the VRAM needed is much lower. If I remember correctly,  it's about 18G per GPU.

If you have any other questions, feel free to let me know.

**Comment by miyajiyuta** [comment]
Dear @tomchen-ctj 
Thank you very much for your detailed response!

I'm truly excited to hear about the upcoming release of the model checkpoints later this month. I'll definitely keep an eye out for that.
